src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -11,6 +11,7 @@ public partial class ByteArrayContent : System.Net.Http.HttpContent
     {
         public ByteArrayContent(byte[] content) { }
         public ByteArrayContent(byte[] content, int offset, int count) { }
+        protected override System.IO.Stream CreateContentReadStream(System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Unfortunetally, this was not regenerated in https://github.com/dotnet/runtime/pull/37494